queries/javascript/rainbow-tags-react.scm

```diff
@@ -0,0 +1,36 @@
+; This query is for highlighting just the JSX tags.
+
+;;; React.js support
+(jsx_element
+  open_tag: (jsx_opening_element
+              "<" @delimiter
+              name: (identifier) @delimiter
+              ">" @delimiter)
+  close_tag: (jsx_closing_element
+               "</" @delimiter
+               name: (identifier) @delimiter
+               ">" @delimiter @sentinel)) @container
+
+(jsx_element
+  open_tag: (jsx_opening_element
+              "<" @delimiter
+              name: (member_expression) @delimiter
+              ">" @delimiter)
+  close_tag: (jsx_closing_element
+              "</" @delimiter
+               name: (member_expression) @delimiter
+              ">" @delimiter @sentinel)) @container
+
+(jsx_self_closing_element
+  "<" @delimiter
+  name: (identifier) @delimiter
+  "/>" @delimiter @sentinel) @container
+
+(jsx_self_closing_element
+  "<" @delimiter
+  name: (member_expression) @delimiter
+  "/>" @delimiter @sentinel) @container
+
+(jsx_expression
+  "{" @delimiter
+  "}" @delimiter @sentinel) @container
```

Comment:
Are you sure you want this pattern? It will highlight the curly braces in expressions like this as well:

```
<div style={{ display: 'flex' }}>
```

Reply:
Oh thanks, good catch!